src/components/ec-input-field/ec-input-field.vue

```diff
@@ -204,9 +204,8 @@ function focus() {
 }
 
 watchEffect(() => {
-  // Hack: since we enabled @vue/compat we occasionally run into issues that the inputModel and value of the input are getting out of sync
+  // Hack: since Vue 3 we occasionally run into issues that the inputModel and value of the input are getting out of sync.
```

Comment:
Really strange behavior, is there any task for investigate it?

Reply:
Nope, I'll create one. It happens in our components only so I believe it has something to do with our code. The ec-dropdown uses ec-input-field in readonly mode to display the selected value. The problems start when we render the ec-dropdown with initial `modelValue` Without this hack, the input gets rendered empty instead of having the value in it. When checking the components in devtools, we see that inputModel in ec-input-field and modelValue in ec-dropdown are set and have the same value, but the input is empty ... I couldn't simulate it in https://sfc.vuejs.org/ so I believe there must be something going on with ec-dropdown and ec-input-field.

Reply:
That is the problem that VTU can't provide the native browser experience. I thought recently that maybe it is worth to use some headless solution for the unit tests too.

Reply:
Yeap, it may be worth trying, both, Storybook and Cypress, have introduced component testing that you can execute in the browser.